From 456fb82378ec975413c30572fcfe7d3981e67c37 Mon Sep 17 00:00:00 2001
From: Yuqing Zhu <carol.zhu@nxp.com>
Date: Wed, 27 Sep 2017 01:37:53 +0800
Subject: [PATCH] wayland-egl.pri: Fix waylandeglclientbufferintegration
 compiling error [YOCIMX-1785]

With wayland backend build, there is a build break.

In included file: eglplatform.h: fatal error: X11/Xlib.h: No such file

The EGL CFLAGS are missing, which should be gained when doing pkg config.

Add egl into QMAKE_USE_PRIVATE to gain -DLINUX -DEGL_API_FB -DWL_EGL_PLATFORM.

Signed-off-by: Yuqing Zhu <carol.zhu@nxp.com>
---
 src/hardwareintegration/compositor/wayland-egl/wayland-egl.pri | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/hardwareintegration/compositor/wayland-egl/wayland-egl.pri b/src/hardwareintegration/compositor/wayland-egl/wayland-egl.pri
index df3aead..e20c680 100644
--- a/src/hardwareintegration/compositor/wayland-egl/wayland-egl.pri
+++ b/src/hardwareintegration/compositor/wayland-egl/wayland-egl.pri
@@ -1,6 +1,6 @@
 INCLUDEPATH += $$PWD
 
-QMAKE_USE_PRIVATE += wayland-server wayland-egl
+QMAKE_USE_PRIVATE += egl wayland-server wayland-egl
 
 QT += egl_support-private
 
-- 
2.7.4

